README.md

```diff
@@ -145,7 +145,7 @@ wget https://github.com/aurora-is-near/aurora-engine/releases/download/latest/ma
 ### Installing the Aurora CLI tool
 
 ```sh
-npm install -g aurora-is-near/aurora-cli
+npm install -g @auroraisnear/cli
 ```
```

Comment:
Actually this whole CLI section should be remove as this library is deprecated.

Reply:
Can we confirm whether "@auroraisnear" is us or not? If it isn't, then we are exposing people to dangerous tech.

Reply:
It works for now. I've checked it out. We could rework the section when `aurora-cli-rs` will be ready.

Reply:
Because it works, doesn't mean it is ours. Its dangerous to proceed with this section. I would remove all mention of the CLI as it is deprecated.

Reply:
Ok. We will modify the README after the `aurora-cli-rs` release.